sdk/src/connections/target/buffer_processor.cpp

```diff
@@ -211,6 +211,7 @@ aditof::Status BufferProcessor::processBuffer(uint16_t *buffer = nullptr) {
     unsigned int buf_data_len;
     uint8_t *pdata;
     dev = m_inputVideoDev;
+    uint8_t *pdata_user_space;
```

Comment:
why is this needed?

Reply:
In case we copy the values on pdata in a new pointer, and than pass it on to tofiCompute, we can have a much faster processing time.

Reply:
I would initialize this to nullptr.